File: finah-desktop/Klassen/finah_desktop/SuperklasseAandoeningPathologie.java
package finah_desktop;

public abstract class SuperklasseAandoeningPathologie {
	private int Id;
	private String Omschrijving;

	public SuperklasseAandoeningPathologie() {

	}

	public SuperklasseAandoeningPathologie(String omschrijving) {
		this.Omschrijving = omschrijving;
	}

	public int getId() {
		return Id;
	}

	public void setId(int id) {
		this.Id = id;
	}



	public String getOmschrijving() {
		return Omschrijving;
	}

	public void setOmschrijving(String omschrijving) {
		this.Omschrijving = omschrijving;
	}
}

File: finah-web/PHP/Models/SuperKlasseAandoeningPathologie.php
<?php

    /**
     * Created by PhpStorm.
     * User: Gert
     * Date: 25/03/2015
     * Time: 23:10
     */
    abstract class SuperKlasseAandoeningPathologie
    {
        public $Id;
        public $Omschrijving;

        public function __construct($id = null, $omschrijving = null)
        {
            $this->Id = $id;
            $this->Omschrijving = $omschrijving;
        }

        public function getId()
        {
            return $this->Id;
        }

        public function setId($Id)
        {
            $this->Id = $Id;
        }

        public function getOmschrijving()
        {
            return $this->Omschrijving;
        }

        public function setOmschrijving($Omschrijving)
        {
            $this->Omschrijving = $Omschrijving;
        }

    }

?>